### ISSUE TITLE
askCard with A custom text description

### ISSUE THREAD

**Issue by wennx96** [kellyelton/OCTGN#1362]
the askCard dialog only show card image, but if you have two or more same cards in different group or condition, it will be hard to recognise them. So I think it will be better to allow us to add a custom text description for every card in the askCard dialog

**Comment by db0** [comment]
Duplicate of #1298

**Comment by wennx96** [comment]
@db0 not the same question, we only need a text description for a card, not the full status. 
what i need is just like the picture below

![image](https://cloud.githubusercontent.com/assets/8449304/5887948/bab7133e-a425-11e4-838d-8af8ae60a39e.png)

**Comment by db0** [comment]
You mean just the index in a pile? This is anyway shown by the order of the cards (first card is top, then second etc). This is effectively a subset of the one I linked.

**Comment by wennx96** [comment]
@db0 yes,i think that should be more useful.You can provide group name, group index, card condition or everything you want as a list of strings
